### PR TITLE
{lang}[foss/2020a] rstudio v1.3.959

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3636ffdde6e384bab8bdb237508e999b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/f76f04bb0f7991e5b02a5548f0ea9338 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/7c3ad0c6a7308a7fb8af11cfe52da441 for a full test report.

File: easybuild/easyconfigs/r/rstudio/rstudio-1.3.959-foss-2020a-Java-11-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
+version = '1.3.959'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+
+homepage = 'https://www.rstudio.com/'
+description = """This RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R."""
```
Comment (verdurin) [inline easybuild/easyconfigs/r/rstudio/rstudio-1.3.959-foss-2020a-Java-11-R-4.0.0.eb:9]: @branfosj Why have you removed the information on how to start the server?

Comment (branfosj) [reply easybuild/easyconfigs/r/rstudio/rstudio-1.3.959-foss-2020a-Java-11-R-4.0.0.eb]: I based this off our local copy and that does not have the information on how to start the server. I've just updated to put the information back in.

Comment (verdurin) [reply easybuild/easyconfigs/r/rstudio/rstudio-1.3.959-foss-2020a-Java-11-R-4.0.0.eb]: @branfosj thanks.

**Comment by verdurin** [comment]
Going in, thanks @branfosj!

**Pull request merged**